content-src/components/Header/Header.js

```diff
@@ -8,7 +8,8 @@ const Header = React.createClass({
   render() {
     const props = this.props;
     const currentRoute = props.currentRoute || {};
-    return (<header className="head">
+    return (<header className="head" hidden={props.currentRoute.path === "/"}>
```

Comment:
btw this is a pretty hacky solution, we should probably refactor to put this header component inside the `Timeline` component since it is no longer needed in both